runtime/libia2/include/ia2_internal.h

```diff
@@ -210,19 +214,13 @@ asm(".macro movz_shifted_tag_x18 tag\n"
         "mov ia2_stackptr_" #i "@GOTTPOFF(%%rip),%%r11\n"                      \
         /* check that stack pointer holds NULL */                              \
         "cmpq $0x0,%%fs:(%%r11)\n"                                             \
-        "je .Lfresh_init" #i "\n"                                              \
+        "je .Lfresh_init%=" #i "\n"                                            \
```

Comment:
What does this `%=` syntax do?

Reply:
Creates a fresh unique label (this asm block is now duplicated and labels are a global namespace).

Reply:
This might be worth commenting because I couldn't find any docs for this syntax in a cursory google search.

Reply:
Ohh, this is an inline ASM thing rather than an assembler feature: https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html#Special-format-strings